### PR TITLE
Fix: french typos

### PR DESCRIPTION
**Pull request by omahs**
Fix: french typos

### CODE REVIEW

**Comment by omahs** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by brad-decker — DISMISSED** [review]

**Comment by brad-decker** [comment]
@GuillaumeRx I think this is more of a Marketing question than a question about the translation, are we treating "Beta" as a proper name, IE **not** the french word Bêta but retaining the "english" Beta... I would assume we are treating beta as a qualifier word not a part of the name of the distribution due to other areas in the translation having the proper accent on the character, even when presented capitalized as part of the name. That's why I approved but I thought i'd tag a native french speaker who might understand the subtleties of translation with names, etc and see if we should be changing MetaMask Bêta -> MetaMask Beta or accepting this PR

**Comment by omahs** [comment]
I just noticed that "Supprimer" was mispelled, so I added a commit.

**Comment by GuillaumeRx** [comment]
> I think this is more of a Marketing question than a question about the translation, are we treating "Beta" as a proper name, IE **not** the french word Bêta but retaining the "english" Beta... I would assume we are treating beta as a qualifier word not a part of the name of the distribution due to other areas in the translation having the proper accent on the character, even when presented capitalized as part of the name. That's why I approved but I thought i'd tag a native french speaker who might understand the subtleties of translation with names, etc and see if we should be changing MetaMask Bêta -> MetaMask Beta or accepting this PR

@brad-decker This is a tuff question, I would lean more thorwards keeping the english version of `Beta` in the names like `MetaMask Beta`. Most of the softwares I use don't translate words like `Beta` or `Desktop` in their names and the "french community" is pretty used to have the software names non-translated. Most of them get out of this situation by sayin `Download the beta version of [...]` instead of `Download [...] Beta` and if the name includes `Beta` they leave it as it is.

### Brave Beta
image

File: app/_locales/fr/messages.json [hunk 1]
```diff
@@ -370,7 +370,7 @@
     "message": "Conditions d’utilisation"
   },
   "betaMetamaskVersion": {
-    "message": "Version MetaMask Beta"
+    "message": "Version MetaMask Bêta"
```
Comment (GuillaumeRx) [inline app/_locales/fr/messages.json:373]:
```suggestion
    "message": "MetaMask version bêta"
```

File: app/_locales/fr/messages.json [hunk 2]
```diff
@@ -360,7 +360,7 @@
...
   },
   "betaMetamaskDescriptionExplanation": {
-    "message": "Utilisez cette version pour tester les futures fonctionnalités avant leur lancement. Votre utilisation et vos commentaires nous aident à développer la meilleure version possible de MetaMask. Votre utilisation de MetaMask Beta est conditionnée par notre norme $1, ainsi que par notre $2. Il s’agit de la version bêta, présentant un risque de bogues plus élevé. En continuant, vous acceptez et reconnaissez ce risque, ainsi que ceux mentionnés dans nos Conditions d’utilisation et nos Conditions bêta.",
+    "message": "Utilisez cette version pour tester les futures fonctionnalités avant leur lancement. Votre utilisation et vos commentaires nous aident à développer la meilleure version possible de MetaMask. Votre utilisation de MetaMask Bêta est conditionnée par notre norme $1, ainsi que par notre $2. Il s’agit de la version bêta, présentant un risque de bogues plus élevé. En continuant, vous acceptez et reconnaissez ce risque, ainsi que ceux mentionnés dans nos Conditions d’utilisation et nos Conditions bêta.",
```
Comment (GuillaumeRx) [inline app/_locales/fr/messages.json:363]:
```suggestion
    "message": "Utilisez cette version pour tester les futures fonctionnalités avant leur lancement. Votre utilisation et vos commentaires nous aident à développer la meilleure version possible de MetaMask. Votre utilisation de la version bêta de MetaMask est conditionnée par notre norme $1, ainsi que par notre $2. Il s’agit de la version bêta, présentant un risque de bogues plus élevé. En continuant, vous acceptez et reconnaissez ce risque, ainsi que ceux mentionnés dans nos Conditions d’utilisation et nos Conditions bêta.",
```

File: app/_locales/fr/messages.json [hunk 3]
```diff
@@ -260,7 +260,7 @@
     "description": "The name of the application"
   },
   "appNameBeta": {
-    "message": "MetaMask Beta",
+    "message": "MetaMask Bêta",
```
Comment (GuillaumeRx) [inline app/_locales/fr/messages.json:263]:
```suggestion
    "message": "MetaMask Beta",
```

**Review by GuillaumeRx — COMMENTED** [review]
Left some suggestions

**Comment by omahs** [comment]
@GuillaumeRx I just updated according to you suggestions!

**Review by GuillaumeRx — APPROVED** [review]
Looks good to me ! Will leave the last word to @brad-decker 😄

**Review by brad-decker — APPROVED** [review]

**Comment by brad-decker** [comment]
Thank you for the contribution @omahs :D and thanks for reviewing @GuillaumeRx ❤️

**Pull request merged**